tests/functional/lti_certification/v13/core/test_bad_payloads.py

```diff
@@ -10,30 +11,31 @@ class TestBadPayloads:
     """
 
     def test_no_kid_sent_in_jwt_header(
-        self, jwt_headers, make_jwt, test_payload, do_lti_launch
+        self, jwt_headers, make_jwt, test_payload, do_lti_launch, caplog
     ):
         """
         The KID is missing from the header of the JWT (preventing the verification of the signing of the JWT)
         """
         del jwt_headers["kid"]
 
-        response = do_lti_launch(
-            {"id_token": make_jwt(test_payload, jwt_headers)}, status=403
-        )
+        do_lti_launch({"id_token": make_jwt(test_payload, jwt_headers)}, status=403)
 
-        assert response.html
+        assert (
```

Comment:
I'm mostly doing this logging for the benefit of the IMS testing so only asserting it on the functional tests.